spec/services/org/create_last_month_created_plan_service_spec.rb

```diff
@@ -6,6 +6,9 @@
   let(:org) do
     FactoryBot.create(:org, created_at: DateTime.new(2018, 04, 01))
   end
+  let(:org2) do
+    FactoryBot.create(:org)
```

Comment:
we shouldn't need the `FactoryBot.` in these. Ok to leave for now since all of these specs have them. Can clean up later

Reply:
Good to know.
I think I need to spend a bit of time reading about factorybot/rspec best practices/intended usage.

Reply:
me too. There seem to be dozens of different approaches!